.github/workflows/deploy.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   run-without-args:
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 40
```

Comment:
😰 ⏳

Reply:
for some environments, it only take 40s but for other environments, it may take something like ten minutes...